spec/unit/application/lookup_spec.rb

```diff
@@ -546,6 +546,15 @@ def run_lookup(lookup)
         allow(lookup.command_line).to receive(:args).and_return(['g'])
         expect(run_lookup(lookup)).to eql("This is G from facts in facts hash")
       end
+
+      it "fails when a node doesn't have facts" do
+        lookup.options[:node] = Puppet::Node.new("badnode", :facts => nil, :environment => 'production')
+        lookup.options[:render_as] = :s
```

Comment:
Since it fails way before needing to use the `render_as` option, I think we can remove this line from the test.